src/course4/homework/Challenge11And12.java

```diff
@@ -0,0 +1,58 @@
+package course4.homework;
+
+import java.util.Scanner;
+
+public class Challenge11And12 {
+
+    public static void main(String[] args) {
+        System.out.println("This code compares compares two input number and checks if they are withing a specific range");
+
+        Scanner scan = new Scanner(System.in);
+
+        System.out.println("Input the first number:");
+        double firstNumber = scan.nextDouble();
+
+        System.out.println("Input the second number:");
+        double secondNumber = scan.nextDouble();
+
+        boolean firstNumberCheck = (firstNumber >= 1 && firstNumber <=10); //checks if the number is within range (1-10) and returns 'true' or 'false';
+        boolean secondNumberCheck = (secondNumber >10 && secondNumber <=100); //checks if the number is within range (11-100) and returns 'true' or 'false';
+        boolean highFirstNumber = (firstNumber > secondNumber); //checks if the first number is higher than the second;
+        boolean lowFirstNumber = (firstNumber < secondNumber); //checks if the first number is lower than the second;
+
+        //normal challenge requests
+        boolean normalRangeCheck1 = (firstNumber >= 1 && firstNumber <=10); //checks if the number is within range (1-10) and returns 'true' or 'false';
+        boolean normalRangeCheck2 = (secondNumber > 10 && secondNumber <=100); //checks if the number is within range (11-100) and returns 'true' or 'false';
+        boolean normalHighFirstNumber = (firstNumber > secondNumber); //checks if the first number is higher
+        boolean normalHighSecondNumber = (firstNumber < secondNumber); //checks if the first number is lower
```

Comment:
The variables are duplicated in order for me to be able to keep track of things more easily in both my sections which, pretty much, do the same thing but in different ways: one as in the course from Friday and the second one is the way I see that it could be done.

Reply:
I understand, but this is all the more reason you should keep things decoupled. You combined some code but duplicated stuff. This is not nice.